packages/fluent/scss/tooltip/_theme.scss

```diff
@@ -0,0 +1,46 @@
+@use "../core/" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-tooltip--theme() {
+
+    .k-tooltip-wrapper .k-tooltip {
+        @include box-shadow( var( --kendo-tooltip-shadow, #{$kendo-tooltip-shadow} ) );
+    }
+
+    .k-tooltip {
+        @include box-shadow( var( --kendo-tooltip-shadow, #{$kendo-tooltip-shadow} ) );
+
+        --INTERNAL--kendo-tooltip-text: var( --kendo-tooltip-text, #{$kendo-tooltip-text} );
+        --INTERNAL--kendo-tooltip-bg: var( --kendo-tooltip-bg, #{$kendo-tooltip-bg} );
+        --INTERNAL--kendo-tooltip-border: var( --kendo-tooltip-border, #{$kendo-tooltip-border} );
+
+            color: var( --INTERNAL--kendo-tooltip-text, initial );
+            background-color: var( --INTERNAL--kendo-tooltip-bg, initial );
+            border-color: var( --INTERNAL--kendo-tooltip-border, initial );
+
+        .k-callout {
+            color: var( --INTERNAL--kendo-tooltip-bg, initial );
```

Comment:
The callout can be with an inherit color. In this way if the tooltip bg color is changed, the callout would be the same color. This would also allow you to remove the callout bg from the kendo-themes color map